update README to clarify icon & logo duplicates (#886)
home-assistant/brands: README.md

- Shortest side of the image must be not bigger than 256 pixels, 512 pixels for the hDPI version.
- The maximum pixel size for the shortest side of the images is, of course, preferred.

## Using the same image for logo & icon

> If the brand uses the same image for the logo and icon (e.g. if the logo is square), just upload
> the icon images. They will be used as fallback from missing logos.

Reviewer: ```suggestion
If the brand uses the same image for the logo and icon (e.g., if the logo has a square aspect ratio),
only add the icon images. The icon will be used as a fallback for the logo.
```
